inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1586,11 +1586,25 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
-    addSpecificCreator({"PSROIPooling"}, [](const std::shared_ptr<::ngraph::Node> &node,
```

Comment:
why this specific creator is removed?

Reply:
The name stays the same, parameters aren't changed, we're creating CNNLayer, it should work out of the box.